eng/jacoco-test-coverage/pom.xml

```diff
@@ -36,7 +36,7 @@
   </scm>
 
   <properties>
-    <relative.path.to.eng.folder>../..</relative.path.to.eng.folder>
+    <relative.path.to.eng.folder>../../</relative.path.to.eng.folder>
```

Comment:
The parent POM uses this in a way that doesn't expect a trailing `/`

Reply:
Removed trailing `/`